### ISSUE TITLE
Several issues during mintapi exploitation (mintapi not returning any data).

### ISSUE THREAD

**Issue by Mitrij** [mintapi/mintapi#499]
**Describe the bug**
During mintapi exploitation faced several errors and flaws. Tracebacks follows.
Also about timeouts in several code parts, for slow computers waite timeouts in code are too small.
Can you check it and fix? Thanks.

mintapi 2.2

```
Traceback (most recent call last):
  File "/home/scrp/mintapi/mintapi/mintapi/api.py", line 169, in login_and_get_token
    self.status_message = sign_in(
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 365, in sign_in
    handle_different_page_username_password(driver, email)
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 451, in handle_different_page_username_password
    email_input = find_element(
NameError: name 'find_element' is not defined
```
should be driver.find_element instead of just find_element ?

```
Traceback (most recent call last):
  File "/home/scrp/mintapi/mintapi/mintapi/api.py", line 169, in login_and_get_token
    self.status_message = sign_in(
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 394, in sign_in
    account_selection_page(driver, intuit_account)
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 670, in account_selection_page
    WebDriverWait(driver, 20).until(
  File "/usr/local/lib/python3.9/dist-packages/selenium/webdriver/support/wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
```
Looks like 2 seconds not enough for slow computers.

```
Traceback (most recent call last):
  File "/home/scrp/mintapi/mintapi/mintapi/api.py", line 169, in login_and_get_token
    self.status_message = sign_in(
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 365, in sign_in
    handle_different_page_username_password(driver, email)
  File "/home/scrp/mintapi/mintapi/mintapi/signIn.py", line 467, in handle_different_page_username_password
    for username_element in username_elements:
TypeError: 'WebElement' object is not iterable
```

Also, when it finally logs in it does not getting any data (by mint.get_account_data()).

**Comment by iamjli** [comment]
I'm encountering a similar issue on mintapi 2.3

```
>>> mint = mintapi.Mint(username, password)
name 'find_element' is not defined
Traceback (most recent call last):
  File "/Users/johnny/Documents/projects/expenses/venv/lib/python3.6/site-packages/mintapi/signIn.py", line 358, in sign_in
    handle_same_page_username_password(driver, email, password)
  File "/Users/johnny/Documents/projects/expenses/venv/lib/python3.6/site-packages/mintapi/signIn.py", line 437, in handle_same_page_username_password
    email_input = driver.find_element(By.ID, "ius-userid")
  File "/Users/johnny/Documents/projects/expenses/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Users/johnny/Documents/projects/expenses/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/johnny/Documents/projects/expenses/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[id="ius-userid"]"}
  (Session info: chrome=103.0.5060.53)
```

**Comment by burkematthew** [comment]
If you are experiencing `find_element` is not defined, you need to upgrade Selenium on your local machine.

**Comment by iamjli** [comment]
After upgrading to `selenium==4.3.0`, I'm still getting the same error but with a stacktrace:

```
>>> mint = mintapi.Mint(username, password)
name 'find_element' is not defined
Traceback (most recent call last):
  File "/Users/johnny/Documents/projects/expenses/venv3.8/lib/python3.8/site-packages/mintapi/signIn.py", line 358, in sign_in
    handle_same_page_username_password(driver, email, password)
  File "/Users/johnny/Documents/projects/expenses/venv3.8/lib/python3.8/site-packages/mintapi/signIn.py", line 437, in handle_same_page_username_password
    email_input = driver.find_element(By.ID, "ius-userid")
  File "/Users/johnny/Documents/projects/expenses/venv3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 857, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/Users/johnny/Documents/projects/expenses/venv3.8/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 435, in execute
    self.error_handler.check_response(response)
  File "/Users/johnny/Documents/projects/expenses/venv3.8/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[id="ius-userid"]"}
  (Session info: chrome=103.0.5060.53)
Stacktrace:
0   chromedriver                        0x0000000107656079 chromedriver + 4444281
1   chromedriver                        0x00000001075e2403 chromedriver + 3970051
2   chromedriver                        0x000000010727d038 chromedriver + 409656
3   chromedriver                        0x00000001072b32fd chromedriver + 631549
4   chromedriver                        0x00000001072b3581 chromedriver + 632193
5   chromedriver                        0x00000001072e54f4 chromedriver + 836852
6   chromedriver                        0x00000001072d07bd chromedriver + 751549
7   chromedriver                        0x00000001072e3221 chromedriver + 827937
8   chromedriver                        0x00000001072d0683 chromedriver + 751235
9   chromedriver                        0x00000001072a6a45 chromedriver + 580165
10  chromedriver                        0x00000001072a7a95 chromedriver + 584341
11  chromedriver                        0x000000010762755d chromedriver + 4253021
12  chromedriver                        0x000000010762c3a1 chromedriver + 4273057
13  chromedriver                        0x000000010763116f chromedriver + 4292975
14  chromedriver                        0x000000010762cdea chromedriver + 4275690
15  chromedriver                        0x000000010760654f chromedriver + 4117839
16  chromedriver                        0x0000000107646ed8 chromedriver + 4382424
17  chromedriver                        0x000000010764705f chromedriver + 4382815
18  chromedriver                        0x000000010765d8d5 chromedriver + 4475093
19  libsystem_pthread.dylib             0x00007ff81386a4f4 _pthread_start + 125
20  libsystem_pthread.dylib             0x00007ff81386600f thread_start + 15
```

**Comment by CalvinChanCan** [comment]
I'm also experiencing the same issue. It seems like the issue is `find_element` is missing `driver`. After changing `find_element` to `driver.find_element`, it works as expected again.
```
        email_input = find_element(
            By.CSS_SELECTOR,
            '#ius-identifier, [data-testid="IdentifierFirstIdentifierInput"]',
        )
```

```
        email_input = driver.find_element(
            By.CSS_SELECTOR,
            '#ius-identifier, [data-testid="IdentifierFirstIdentifierInput"]',
        )
```

**Comment by mrooney** [comment]
#502 is merged and released as 2.4! If there are any remaining issues (such as the 2 seconds), please open as individual issues so they can be better tracked. Thanks!

**Comment by iamjli** [comment]
Works, thank you!
